src/frontend_analysis/IR_analysis/InterfaceInfer.cpp

```diff
@@ -613,7 +640,7 @@ DesignFlowStep_Status InterfaceInfer::Exec()
 
                if(!isRead && !isWrite)
                {
-                  if(interface_type == "m_axi")
+                  if(interface_type == "m_axi" || interface_type == "fifo")
```

Comment:
This will not work correctly with ac_types since type bitwidth differs from struct bitwidth and can only be inferred from load/store operations. That is why the check was there. Have you tried one of those use cases?